### ISSUE TITLE
Assume Role With SAML and Update Credentials File

### ISSUE THREAD

**Issue by Joe-Zer0** [aws/aws-cli#8277]
### Describe the feature

I would like to propose a function similar to `aws sts assume-role-with-saml` but instead of returning the credentials to stdout, it will update the credentials file.  There is already precedence for the aws cli updating files via the `aws codeartifact login` command.  I think this would be a nice quality of life feature to save clients copying and pasting the credentials or writing their own code to update the credentials file.

### Use Case

When I log in with SAML, I have to copy and paste the returned credentials into the credentials file, or I have to write code to update the credentials file.  It would be nice to have a native function to update the credentials file for me.

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### CLI version used

aws-cli/2.13.25 Python/3.11.5 Windows/10 exe/AMD64 prompt/off

### Environment details (OS name and version, etc.)

Windows 10

**Comment by tim-finnigan** [comment]
Hi @Joe-Zer0 thanks for reaching out. Have you tried running the [configure](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/configure/index.html#cli-aws-configure) or [configure set](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/configure/set.html) commands? You can update the credentials file using those commands.

**Comment by Joe-Zer0** [comment]
Yes, but that requires stitching the two commands together. I believe this is a common enough process it could warrant an ease of use function.

**Comment by tim-finnigan** [comment]
Thanks for following up, I think we can leave this open for further tracking and consideration. Others can 👍 the issue if also interested in this.
